_pytest/python.py

```diff
@@ -1838,34 +1838,51 @@ def __init__(self, pyfuncitem):
         self.fixturename = None
         #: Scope string, one of "function", "class", "module", "session"
         self.scope = "function"
-        self._funcargs  = {}
-        self._fixturedefs = {}
+        # rename both attributes below because their key has changed; better an attribute error
+        # than subtle key misses; also backward incompatibility
```

Comment:
note to self: not sure this is a backward incompatibility that should be noted at all in the CHANGELOG, after all is an internal attribute rename